hushline/model.py

```diff
@@ -40,6 +40,7 @@ class OrganizationSetting(Model):
     BRAND_LOGO = "brand_logo"
     BRAND_NAME = "brand_name"
     BRAND_PRIMARY_COLOR = "brand_primary_color"
+    DIRECTORY_INTRO = "directory_intro"
```

Comment:
It might be worth adding a suffix like `_html` or `_markdown` or `_text` to make it more clear to us when we use wit what it's supposed to be. I assume we want `html` but would need to use something like [`bleach`](https://pypi.org/project/bleach/) to sanitize it to a same minimal subset of HTML.

Reply:
sounds good but lets hold until we can rationalize applying this pattern holistically.